src/styles/product-card.scss

```diff
@@ -0,0 +1,89 @@
+@import 'fonts';
+@import 'stars';
+@import 'variables';
+
+.product-card {
+  box-sizing: border-box;
```

Comment:
you already have it in the `body`. move it from `body` to `*` so it works for all cases and don't duplicate
```suggestion
```

Reply:
AI Mentor suggests not to use "*" selector and rejects PR if I use it